scripts/kind-two-node-prometheus-cluster.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    kubeadmConfigPatches:
+      - |
+        apiVersion: kubeadm.k8s.io/v1beta2
+        kind: ClusterConfiguration
+        metadata:
+          name: config
+  - role: worker
+    extraPortMappings: # Host machine to Prometheus service reachability
+      - containerPort: 30900
+        hostPort: 9090
+        listenAddress: "0.0.0.0" # Optional, defaults to "0.0.0.0"
+        protocol: tcp # Optional, defaults to tcp
```

Comment:
where did this file come from?

Reply:
We were enabling prometheus port-forwarding, by default on all KIND clusters. This caused port collision issues when I tried to run multiple e2e tests on the same machine. I don't see this being an issue for our CI, but it could be an issue for local development. I had to create a separate KIND config, one with Prometheus port forwarding, and remove the port forwarding from the default one. Then I use the Prometheus specific one only when the `ENABLE_PROMETHEUS` flag is set `true`.